schemas/v3.1/schema-base.yaml

```diff
@@ -1,3 +1,4 @@
+description: A dialect to use for schemas within OpenAPI documents
```

Comment:
This isn't a dialect schema. It describes an OpenAPI document where the schemas all use the OpenAPI Schema dialect.